packages/react/cypress/component/auto/form/AutoFormDefaultValues.cy.tsx

```diff
@@ -26,6 +26,46 @@ describeForEachAutoAdapter("AutoForm - Default model field values", ({ name, ada
     cy.get(`input[name="part.notes"]`).should("have.value", "");
   });
 
+  it("can set default values in props for excluded fields and send them to the server", () => {
+    cy.mountWithWrapper(
+      <AutoForm action={api.part.create} defaultValues={{ part: { name: "test record" } }} exclude={["name"]} />,
+      wrapper
+    );
+
+    cy.get(`input[name="part.count"]`).should("have.value", "");
+    cy.get(`input[name="part.notes"]`).should("have.value", "");
```

Comment:
Could you add one check to make sure the `name` field doesn't exist?

Reply:
Updated to include these cases into storybook and to check that the name input is gone in the Cypress test